### ISSUE TITLE
Triger DB plugin install from UI

### ISSUE THREAD

**Issue by Akirtovskis** [ledokku/ledokku#268]
# Feature request

Currently to enable new db you need to run a command from server

Would be nice to have this working from UI 

## Describe the solution you'd like

- Create new command in server/dokku lib 
- Use some neat Chakra UI loading and feedback components 
- Voila

**Comment by pradel** [comment]
> Currently to enable new db you need to run a command from server

Do you mean enabling the plugin?

**Comment by Akirtovskis** [comment]
Yeah, instead of displaying the `terminal` window with ssh command I think we could just add to be triggered form UI, right?

**Comment by pradel** [comment]
I think there was some permissions issues while doing this, let's try again but I am not sure it's possible :(
